include/verilated_vpi.cpp

```diff
@@ -1323,285 +1350,294 @@ void vpi_put_delays(vpiHandle /*object*/, p_vpi_delay /*delay_p*/) { _VL_VPI_UNI
 
 // value processing
 
-void vpi_get_value(vpiHandle object, p_vpi_value value_p) {
```

Comment:
For clarity, I recommend enabling ignore whitespace in diff settings as the indentation had to be changed.